auxiliary/crash-handler/src/CommandLine.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+/// Parse a command line string from chatterino into its arguments.
+///
+/// The command line arguments are joined by '+'. A plus is escaped by an
+/// additional plus ('++' -> '+').
+std::vector<std::wstring> splitChatterinoArgs(const std::wstring &args);
```

Comment:
Rename to `splitEncodedArgs` or `splitEncodedChatterinoArgs`
Parameter name should be updated accordingly, the first line of the comment should include something about that it's encoded as well